<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_hello">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hello_002">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_html_001">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_html_002">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_jpg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gif">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
